test/nonregression/test_run_pipelines.py

```diff
@@ -699,6 +699,48 @@ def test_run_SpatialSVM(cmdopt):
     # Remove data in out folder
     clean_folder(join(root, 'out', 'caps'), recreate=True)
 
+def test_run_T1Linear(cmdopt):
+    from os.path import dirname, join, abspath
+    import shutil
+    from clinica.pipelines.t1_linear.t1_linear_pipeline import T1Linear
+    import nibabel as nib
+    import numpy as np
+
+    working_dir = cmdopt
+    root = dirname(abspath(join(abspath(__file__), pardir)))
+    root = join(root, 'data', 'T1Linear')
+
+    # Remove potential residual of previous UT
+    clean_folder(join(working_dir, 'T1Linear'))
+    clean_folder(join(root, 'out', 'caps'), recreate=False)
+    shutil.copytree(join(root, 'in', 'caps'), join(root, 'out', 'caps'))
+
+    # Instantiate pipeline
+    pipeline = T1Linear(
+        bids_directory=join(root, 'in', 'bids'),
+        caps_directory=join(root, 'out', 'caps'),
+        tsv_file=join(root, 'in', 'subjects.tsv'),
+        base_dir=join(working_dir, 'T1Linear'),
+    )
+    pipeline.build()
```

Comment:
pipeline is automatically build when it is `run()` so this line is not necessary

Reply:
Indeed. But this is also the case for all other `test_run_Pipeline()`.

@mdiazmel : Could you take the opportunity of this PR to remove `pipeline.build()` for `test_run_pipelines.py`

Reply:
Done!